14-draft.json

```diff
@@ -1058,6 +1058,31 @@
         "type": "string"
       }
     },
+    "links": {
+      "description": "Arbitrary links that you'd like to share",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The link name.",
+            "type": "string"
+          },
+          "description": {
+            "description": "An extra field for a more detailed description of the link.",
+            "type": "string"
+          },
+          "url": {
```

Comment:
Just some bike shedding: Why URL and not URI?

Reply:
Consistency with the rest of the schema.

Reply:
Also, I think we actually do want an URL and not an URI.